### DISCUSSION TITLE
help me with mscloader ols

### DISCUSSION THREAD

**Discussion by Yourdrunkfather** [piotrulos/MSCModLoader#180] [Help me] [answered]
when i open msc folder in mscpatcher it shows me an error pls help (sorry but im from poland)
![help](https://github.com/piotrulos/MSCModLoader/assets/144143159/e5bb3bd6-8d5b-4f1d-98bf-c4b031bdd876)

**Comment by LeFlumi** [comment, marked as answer]
have you tried reinstalling it?

**Reply by Yourdrunkfather** [reply]
yes i tried it so much times
